build.zig.zon

```diff
@@ -2,6 +2,7 @@
     .name = "roc",
     .version = "0.0.0",
     .minimum_zig_version = "0.13.0",
+    .fingerprint = 0x9eea22a61075aed2,
```

Comment:
I don't think this should be added yet. This is a 0.14.0 feature. Though I guess it doesn't specifically hurt.